CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ include(GNUInstallDirs)
 
 set(SRC_SHARED
   "support-lib/djinni_common.hpp"
+  "support-lib/project_export.hpp"
```

Comment:
This is used so widely (requiring it to be in the include path) that it really needs a more unique name, like `djinni_project_export.hpp`.